### PR TITLE
chore: include slot in publish to topic log

### PR DESCRIPTION
**Pull request by nflaig**
**Motivation**

Make publish to topic logs searchable by slot

**Description**

Include slot in publish to topic log

### CODE REVIEW

File: packages/beacon-node/src/network/network.ts [hunk 1]
```diff
@@ -408,7 +408,7 @@ export class Network implements INetwork {
     };
     const sentPeers = await this.core.publishGossip(topicStr, messageData, opts);
 
-    this.logger.verbose("Publish to topic", {topic: topicStr, sentPeers});
+    this.logger.verbose("Publish to topic", {topic: topicStr, sentPeers, slot: this.clock.currentSlot});
```
Comment (nflaig) [inline packages/beacon-node/src/network/network.ts]: Not every message includes a  slot, e.g. voluntary  exit or bls to execution change, that's why I opted to use the clock slot instead when the message was published.

To clarify this, we could also do `clockSlot: this.clock.currentSlot` or `currentSlot: this.clock.currentSlot`, but curious what others think

Comment (wemeetagain) [reply packages/beacon-node/src/network/network.ts]: Personally think the clarification of `clockSlot` or other might be helpful here.

Comment (nflaig) [reply packages/beacon-node/src/network/network.ts]: `currentSlot` seems to be the more commonly used, and I think we only use it for the current slot in terms of time-based, whereas `slot` could be message slot or time-based slot

**Review by wemeetagain — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.18.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.18.0) :tada:

**Pull request merged**